test/evp_pkey_dhkem_test.c

```diff
@@ -0,0 +1,871 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "testutil.h"
+
+#ifndef OPENSSL_NO_EC
```

Comment:
We basically seem to skip everything if EC is not enabled. Why not avoid building this file in build.info if there is no ec. Similarly skip the whole thing in 30-test_evp_pkey_dhkem.t if no-ec. That way we can avoid all the ifdefs in this file.

Reply:
ok